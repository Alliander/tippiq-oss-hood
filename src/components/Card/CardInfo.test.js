import React from 'react';
import renderer from 'react-test-renderer';
import Element from './CardInfo';

test('Renders', () => {
  const component = renderer.create(
    <Element><div>x</div></Element>
  );
  const json = component.toJSON();
  expect(json).toMatchSnapshot();
});

