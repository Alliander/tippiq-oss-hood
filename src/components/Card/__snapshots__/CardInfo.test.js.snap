exports[`test Renders 1`] = `
<div
  className="undefined ">
  <span
    className={undefined} />
  <span
    className={undefined}>
    <div>
      x
    </div>
  </span>
</div>
`;
